samza-api/src/main/java/org/apache/samza/operators/windows/Windows.java

```diff
@@ -24,8 +24,11 @@
 import org.apache.samza.operators.triggers.Trigger;
 import org.apache.samza.operators.triggers.Triggers;
 import org.apache.samza.operators.windows.internal.WindowInternal;
+import org.apache.samza.operators.windows.internal.WindowType;
 
 import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.function.BiFunction;
 import java.util.function.Function;
```

Comment:
This comment pertains to Line 82 (Github is not allowing me to add a comment there :( )

QQ (perhaps orthogonal to current patch): The javadoc in Line 82 - Does it "require" an early trigger to be defined by the user? Looks like there is no default behavior. If that is the intended case, how are we enforcing this on the user? Shouldn't Trigger be a parameter in globalWindow() ?

I could be wrong as I just started looking into the api code. Please correct if I am wrong. Thanks!